internal/pkg/cli/env_delete.go

```diff
@@ -348,6 +372,80 @@ func (o *deleteEnvOpts) ensureRolesAreRetained() error {
 	return nil
 }
 
+// emptyBuckets returns nil if buckets were deleted successfully. Otherwise, returns the error.
+func (o *deleteEnvOpts) emptyBuckets() error {
+	envStack, err := o.rg.GetResources(&resourcegroupstaggingapi.GetResourcesInput{
+		ResourceTypeFilters: []*string{aws.String("cloudformation")},
```

Comment:
(Just to mention `aws.StringSlice` as an option in case you didn't know, not a request for code change)

Alterntatively ⬇️ 
```go
		ResourceTypeFilters: aws.StringSlice([]string{"cloudformation}),
```